x/schema/queue.go

```diff
@@ -14,6 +14,7 @@ func GenerateCreateTableQuery(queueName string) string {
 	(
 		id             UUID        DEFAULT gen_random_uuid() NOT NULL PRIMARY KEY,
 		created_at     TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL,
+		delayed_until  TIMESTAMPTZ                           NULL,
```

Comment:
This is unfortunately breaking change. In this case, I hope not many people use the library yet :smile:

Reply:
hmmm, agree, not a big deal add new column manualy. but yes this SHOULD be very highlighted in release notes. And maybe good time to think about upgrade mechanism. somethink like pgqInit function that check chenges in table and propse/do the necesarry changes. This is likely not last change in the DB